.travis.yml

```diff
@@ -72,6 +72,20 @@ jobs:
     - MB_ML_VER=2014
     - PLAT=aarch64
     - DOCKER_TEST_IMAGE=multibuild/xenial_arm64v8
+  - os: linux
+    env:
+    - MB_PYTHON_VERSION=3.9
+  - os: linux
+    env:
+    - MB_PYTHON_VERSION=3.9
+    - PLAT=i686
+  - os: linux
+    env:
+    - MB_PYTHON_VERSION=3.10
+  - os: linux
+    env:
+    - MB_PYTHON_VERSION=3.10
+    - PLAT=i686
```

Comment:
Sticking with manylinux1 for now in order to build on the oldest supported glibc version.  Per [PEP-600](https://peps.python.org/pep-0600/) this is equivalent to a build with glibc 2.5.  Given glibc backward compatibility rules this should work for most glibc versions going forward.